config/webpack/webpack.config.js

```diff
@@ -39,29 +101,59 @@ module.exports = merge(shims, {
             ],
             plugins: [
               '@babel/plugin-proposal-object-rest-spread',
-              'angularjs-annotate'
+              'angularjs-annotate',
+              ...ifDevelopment('react-refresh/babel')
             ]
           }
         }]
       },
+      {
+        test: /\.(woff2?|eot|ttf|otf)(\?.*)?$/,
+        loader: 'url-loader',
+        options: {
+          limit: 10000,
+          name: '[name].[hash:7].[ext]',
+          outputPath: 'fonts/'
+        }
+      },
       {
         test: /\.(png|jpe?g|gif|svg)$/,
         use: [
           {
-            loader: 'file-loader',
+            loader: 'url-loader',
             options: {
-              name: '[name]-[hash].[ext]',
+              limit: 10000,
+              name: '[name]-[hash:7].[ext]',
               outputPath: 'images/'
             }
           }
         ]
+      },
+      {
+        test: /\.css$/,
+        use: styleLoaders
+      },
+      {
+        test: /\.less$/,
+        use: [
+          ...styleLoaders,
+          {
+            loader: 'less-loader'
+          }
+        ]
       }
     ]
   },
   plugins: [
+    ...ifProduction(new MiniCssExtractPlugin({
```

Comment:
Another pattern I've been using is with [`lodash.compact`](https://lodash.com/docs/4.17.15#compact):

```js
plugins: compact( [
  isProduction && new MiniCssExtractPlugin(),
  new webpack.DefinePlugin(),
  ! isProduction && new ReactRefreshWebpackPlugin(),
] )
```

Up to you which you prefer — personally I like not having helper internal functions and relying on Lodash instead. :-)

It would also simplify if we treat `isProduction` / `isDevelopment` as one constant, so that `! isProduction` would always be `development`?

Reply:
I switched to the `compact` approach, I'm pretty agnostic, but like to try new things!

`isProduction` is not the opposite of `isDevelopment`, because of test environment (the react refresh stuff broke stuff when loaded in test environment iirc).